core/frontend-stubs/maven-frontend-stub/pom.xml

```diff
@@ -96,6 +96,10 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
```

Comment:
Why is the dependency here necessary?

Reply:
Because otherwise `org.eclipse.sw360.antenna.frontend.stub.mojo.DependencyTest` failed.